playbook.yml

```diff
@@ -14,4 +14,4 @@
      - zimon/precheck
      - zimon/node
      - zimon/cluster
-     - zimon/postcheck
```

Comment:
Ignore this file for checkin . don't do git add .  . run git add <your_modifed_file>